tokio-console/src/config.rs

```diff
@@ -163,11 +206,43 @@ impl Config {
     }
 }
 
+fn merge_view_options(base: &mut ViewOptions, command_line: ViewOptions) {
+    if command_line.no_colors.is_some() {
+        base.no_colors = command_line.no_colors;
+    }
```

Comment:
nit, take it or leave it: i think we could potentially simplify these using [`Option::or`](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.or):
```suggestion
   base.no_colors = command_line.no_colors.or(base.no_colors.take());
```